packages/language-server/types/astro-jsx.d.ts

```diff
@@ -447,7 +447,7 @@ declare namespace astroHTML.JSX {
 		classid?: string | undefined | null;
 		cols?: number | undefined | null;
 		colspan?: number | undefined | null;
-		content?: string | undefined | null;
+		content?: string | URL | undefined | null;
```

Comment:
Some users tend to use URL for this property (especially for like og:image), since we convert those to string in Astro, it should be accepted by the language-server too